userspace/libsinsp/sinsp_filtercheck_event.cpp

```diff
@@ -549,6 +549,13 @@ int32_t sinsp_filter_check_event::parse_field_name(std::string_view val,
 
 		res = extract_arg("evt.rawarg", val, &m_arginfo);
 
+		// These fields will be overridden with correct ones from the current event
+		// while extracting from event.
+		// But we also need to set them at init time to something different than `PT_DYN`
+		// to allow `eval_filter` expressions to be properly compiled.
+		// Note: even if they'll be overridden,
+		// it is safe to guess that eg: `size` param will always be a numeric type.
+		// Of course if we had a `size` param that is string, it would be troublesome.
```

Comment:
i would just remove this since we have already the case of `addr`

Reply:
Yep i am afraid that we forgot once again :laughing: